migration/migrating_3_4/troubleshooting-3-4.adoc

```diff
@@ -19,8 +19,12 @@ include::modules/migration-viewing-migration-crs.adoc[leveloffset=+1]
 include::modules/migration-downloading-logs.adoc[leveloffset=+1]
 include::modules/migration-updating-deprecated-gvks.adoc[leveloffset=+1]
 include::modules/migration-error-messages.adoc[leveloffset=+1]
-include::modules/migration-manually-rolling-back-migration.adoc[leveloffset=+1]
+
+ifeval::["{mtc-version}" >= "1.4"]
```

Comment:
Won't this conditional always be true?

If you only mean for users to perform one of these rollback procedures, I'd add a H2 to the assembly, explain that they need to complete one of these, and then nest the modules under it.

Reply:
RE: conditional being true. No. MTC 1.4 will not be released until Jan. 14.

Re: H2. If I create a header and nest the modules under it, the current version will have H2 followed by only ONE H3 header. Is that allowed? I thought orphan headers were discouraged.

When 1.4 is released, then it would make sense to have H2 header, followed by two H3 sections.

Alternatively, I could create a 1.4 conditional, with H2 header and both modules, and a <1.4 conditional with no header and only the CLI module.

Reply:
I see what you're going for, but by changing the value of the common attributes file now instead of on 1/14, the changes you're trying to hold are going to go live now.

You could also split the PR in half and stage the 1.4+ change to go live Jan. 14 instead of using the conditionals.

I haven't seen a rule discouraging orphaned headers. It's something that I've had to do several times.

Reply:
Good point about the attributes file. I have a separate PR to deal with that, so I should revert the attributes file here. Forgot that it had been changed.

Reply:
I have removed the conditions. This PR will be merged in Jan.